index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
+    </div>
+    <div class="work-language">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+</div>
+    <button type="button" class="button-1">See Project</button>
+</div>
+    <!-- card2 -->
+             
+<section class="work-container1">
+    <div class="img-snapshot2">
+    <img src="images/Snapshoot2.png" alt="">
+    </div>
+    <div class="multipost">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-1">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-1">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-1">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+
+</div>
+    <button type="button" class="button-2">See Project</button>
+
+ </section>
+    <!-- card3 -->
+             
+<section class="work-container2">
+    <div class="img-snapshot3">
+    <img src="images/Snapshoot3.png" alt="">
+    </div>
+    <div class="tonic2">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame-2">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-2">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-2">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+</div>
+  
+    <button type="button" class="button-3">See Project</button>
+
+</section>
+    <!-- card4 -->
+             
+<section class="work-container3">
+    <div class="img-snapshot4">
+    <img src="images/Snapshoot4.png" alt="">
+    </div>
+    <div class="multipost1">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-3">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-3">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-3">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+    </div>
+  
+    <button type="button" class="button-4">See Project</button>
+
+</section>
```

Comment:
- [ ] Please wrap all of this section with a `section` tag, instead of wrapping each card with a `section` container, and put this section outside of the header section, for semantic purposes. You can use something like the following:
```
<body>
   <header> ... Nav elements </header>
   <section> ... Intro section </section>
   <section> ... Works section </section>
   <section> ... About section </section>
</body>
```
instead of having this hierarchy
![image](https://user-images.githubusercontent.com/62290545/165788038-8856d5cb-7f37-4e47-9dc9-3b7491bbce68.png)